config/application.rb

```diff
@@ -32,6 +32,12 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
+    # Active Job Configurations
+    if ENV['REDIS_HOST']
```

Comment:
@whereismyjetpack would `if redis_config.valid?` be better here?

Reply:
yup, i like that much better.